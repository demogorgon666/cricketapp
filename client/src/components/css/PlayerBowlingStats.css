@import url("https://fonts.googleapis.com/css?family=Montserrat:400,500,600&display=swap");

.ball-main {
  display: flex;
  flex-direction: row;
  margin-top: 210px;
  margin-bottom: 40px;
  justify-content: center;
  width: 95vw;
}

.bowling-main-div {
  display: grid;
  grid-template-columns: 32vw 32vw 32vw;
}

#bowling-format-heading {
  background-color: #2980b9;
  color: white;
  padding: 8px;
  margin-top: 0px;
}

.bowling-Test-div {
  border: 1px solid #707070;
  border-top: none;
  margin-left: 70px;
  font-family: "Montserrat", sans-serif;
  width: 300px;
}

.bowling-Odi-div {
  border: 1px solid #707070;
  border-top: none;
  margin-left: 10px;
  width: 300px;
}

.bowling-T20-div {
  border: 1px solid #707070;
  border-top: none;
  margin-left: -10px;
  width: 300px;
}

.bowling-Ipl-div {
  border: 1px solid #707070;
  border-top: none;
  margin-left: 10px;
  margin-top: 25px;
  margin-bottom: 25px;
}

.bowling-stats-info-parent-div {
  display: flex;
  flex-direction: row;
  margin-left: 16px;
  margin-right: 16px;
  border-bottom: 1px solid #707070;
  padding-bottom: 6px;
  padding-top: 10px;
  font-family: "Montserrat", sans-serif;
}

.bowling-stats-info-div {
  display: flex;
  flex-direction: column;
  text-align: center;
  font-family: "Montserrat", sans-serif;
  /* padding: 10px; */
  padding-left: 20px;
}

.bowling-stats-bottom-info-div {
  display: flex;
  flex-direction: column;
  text-align: center;
  font-family: "Montserrat", sans-serif;
  /* padding: 10px; */
  padding-left: 30px;
  margin-bottom: 17px;
}

#ball-stat-span-1 {
  font-weight: 500;
  font-size: 19px;
}

#ball-stat-span-2 {
  font-weight: 400;
  font-size: 13px;
  padding-top: 2px;
}

@media only screen and (max-width: 576px) {
  .ball-main {
    display: flex;
    flex-direction: row;
    margin-top: 230px;
    justify-content: center;
    width: 100vw;
  }

  .bowling-main-div {
    display: grid;
    grid-template-columns: 87vw;
  }

  #bowling-format-heading {
    background-color: #2980b9;
    color: white;
    padding: 4px;
    margin-top: 0px;
    font-size: 16px;
  }

  .bowling-Test-div {
    border: 1px solid #707070;
    border-top: none;
    margin: auto;
    margin-top: 15px;
    font-family: "Montserrat", sans-serif;
  }

  .bowling-Odi-div {
    border: 1px solid #707070;
    border-top: none;
    margin: auto;
    margin-top: 15px;
  }

  .bowling-T20-div {
    border: 1px solid #707070;
    border-top: none;
    margin: auto;
    margin-top: 15px;
  }

  .bowling-Ipl-div {
    border: 1px solid #707070;
    border-top: none;
    margin: auto;
    margin-top: 15px;
  }

  .bowling-stats-info-parent-div {
    display: flex;
    flex-direction: row;
    margin-left: 10px;
    margin-right: 10px;
    border-bottom: 1px solid #707070;
    padding-bottom: 3px;
    padding-top: 3px;
    font-family: "Montserrat", sans-serif;
  }

  .bowling-stats-info-div {
    display: flex;
    flex-direction: column;
    text-align: center;
    font-family: "Montserrat", sans-serif;
    padding-left: 14px;
  }

  .bowling-stats-bottom-info-div {
    display: flex;
    flex-direction: column;
    text-align: center;
    font-family: "Montserrat", sans-serif;
    /* padding: 10px; */
    padding-left: 20px;
  }

  #ball-stat-span-1 {
    font-weight: 500;
    font-size: 13px;
  }

  #ball-stat-span-2 {
    font-weight: 400;
    font-size: 8px;
    padding-top: 3px;
  }
}
