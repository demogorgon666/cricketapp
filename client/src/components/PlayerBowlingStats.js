import React, { Component } from "react";
import "../components/css/PlayerBowlingStats.css";

import {
  getODIBowlerStats,
  getT20BowlerStats,
  getTestBowlerStats
} from "../actions/PlayerAction";

import { connect } from "react-redux";

export class PlayerBowlingStats extends Component {
  componentDidMount() {
    console.log("from mount", this.props.player_id);
    this.props.getODIBowlerStats(this.props.player_id);
    this.props.getT20BowlerStats(this.props.player_id);
    this.props.getTestBowlerStats(this.props.player_id);
    // this.props.getT20BatsmanStats(this.props.player_id);
    // this.props.getTestBatsmanStats(this.props.player_id);
  }

  render() {
    console.log(
      "Component bowling stats ",
      this.props.testBowlerStats["0"].RunsConceded
        ? this.props.testBowlerStats["0"].RunsConceded.length === 0
          ? 0
          : this.props.testBowlerStats["0"].RunsConceded[0].match_type
        : console.log("not found")
    );

    // console.log("Component bowling stats", this.propstestBowlerStats);

    return (
      <div className="ball-main">
        <div className="bowling-main-div">
          <div className="bowling-Test-div">
            <h3 id="bowling-format-heading">Test</h3>
            <div className="bowling-stats-info-parent-div">
              <div className="bowling-stats-info-div">
                <span id="ball-stat-span-1">
                  {this.props.testBowlerStats["0"].Matches
                    ? this.props.testBowlerStats["0"].Matches.length === 0
                      ? "-"
                      : this.props.testBowlerStats["0"].Matches[0]
                          .player_stats_value
                    : console.log("not found")}
                </span>
                <span id="ball-stat-span-2">Matches</span>
              </div>
              <div className="bowling-stats-info-div">
                <span id="ball-stat-span-1">
                  {this.props.testBowlerStats["0"].Innings
                    ? this.props.testBowlerStats["0"].Innings.length === 0
                      ? "-"
                      : this.props.testBowlerStats["0"].Innings[0]
                          .player_stats_value
                    : console.log("not found")}
                </span>
                <span id="ball-stat-span-2">Innings</span>
              </div>
              <div className="bowling-stats-info-div">
                <span id="ball-stat-span-1">
                  {this.props.testBowlerStats["0"].RunsConceded
                    ? this.props.testBowlerStats["0"].RunsConceded.length === 0
                      ? "-"
                      : this.props.testBowlerStats["0"].RunsConceded[0]
                          .player_stats_value
                    : console.log("not found")}
                </span>
                <span id="ball-stat-span-2">Runs Conceded</span>
              </div>
              <div className="bowling-stats-info-div">
                <span id="ball-stat-span-1">
                  {this.props.testBowlerStats["0"].BallsBowled
                    ? this.props.testBowlerStats["0"].BallsBowled.length === 0
                      ? "-"
                      : this.props.testBowlerStats["0"].BallsBowled[0]
                          .player_stats_value
                    : console.log("not found")}
                </span>
                <span id="ball-stat-span-2">Balls bowled</span>
              </div>
            </div>
            <div
              className="bowling-stats-info-parent-div"
              style={{ borderBottom: "none" }}
            >
              <div className="bowling-stats-bottom-info-div">
                <span id="ball-stat-span-1">
                  {this.props.testBowlerStats["0"].TotalWIckets
                    ? this.props.testBowlerStats["0"].TotalWIckets.length === 0
                      ? "-"
                      : this.props.testBowlerStats["0"].TotalWIckets[0]
                          .player_stats_value
                    : console.log("not found")}
                </span>
                <span id="ball-stat-span-2">Wickets</span>
              </div>
              <div className="bowling-stats-bottom-info-div">
                <span id="ball-stat-span-1">
                  {this.props.testBowlerStats["0"].EconomyRate
                    ? this.props.testBowlerStats["0"].EconomyRate.length === 0
                      ? "-"
                      : this.props.testBowlerStats["0"].EconomyRate[0]
                          .player_stats_value
                    : console.log("not found")}
                </span>
                <span id="ball-stat-span-2">Economy rate</span>
              </div>
              {/* <div className="bowling-stats-bottom-info-div">
                <span id="ball-stat-span-1">30.54</span>
                <span id="ball-stat-span-2">Strike Rate</span>
              </div> */}
              <div className="bowling-stats-bottom-info-div">
                <span id="ball-stat-span-1">
                  {this.props.testBowlerStats["0"].FourWickets
                    ? this.props.testBowlerStats["0"].FourWickets.length === 0
                      ? "-"
                      : this.props.testBowlerStats["0"].FourWickets[0]
                          .player_stats_value
                    : console.log("not found")}
                </span>
                <span id="ball-stat-span-2">4 wickets</span>
              </div>
              <div className="bowling-stats-bottom-info-div">
                <span id="ball-stat-span-1">
                  {this.props.testBowlerStats["0"].FiveWickets
                    ? this.props.testBowlerStats["0"].FiveWickets.length === 0
                      ? "-"
                      : this.props.testBowlerStats["0"].FiveWickets[0]
                          .player_stats_value
                    : console.log("not found")}
                </span>
                <span id="ball-stat-span-2">5 wickets</span>
              </div>
            </div>
          </div>
          <div className="bowling-Odi-div">
            <h3 id="bowling-format-heading">ODI</h3>
            <div className="bowling-stats-info-parent-div">
              <div className="bowling-stats-info-div">
                <span id="ball-stat-span-1">
                  {" "}
                  {this.props.odiBowlerStats["0"].Matches
                    ? this.props.odiBowlerStats["0"].Matches.length === 0
                      ? "-"
                      : this.props.odiBowlerStats["0"].Matches[0]
                          .player_stats_value
                    : console.log("not found")}
                </span>
                <span id="ball-stat-span-2">Matches</span>
              </div>
              <div className="bowling-stats-info-div">
                <span id="ball-stat-span-1">
                  {this.props.odiBowlerStats["0"].Innings
                    ? this.props.odiBowlerStats["0"].Innings.length === 0
                      ? "-"
                      : this.props.odiBowlerStats["0"].Innings[0]
                          .player_stats_value
                    : console.log("not found")}
                </span>
                <span id="ball-stat-span-2">Innings</span>
              </div>
              <div className="bowling-stats-info-div">
                <span id="ball-stat-span-1">
                  {this.props.odiBowlerStats["0"].RunsConceded
                    ? this.props.odiBowlerStats["0"].RunsConceded.length === 0
                      ? "-"
                      : this.props.odiBowlerStats["0"].RunsConceded[0]
                          .player_stats_value
                    : console.log("not found")}
                </span>
                <span id="ball-stat-span-2">Runs Conceded</span>
              </div>
              <div className="bowling-stats-info-div">
                <span id="ball-stat-span-1">
                  {this.props.odiBowlerStats["0"].BallsBowled
                    ? this.props.odiBowlerStats["0"].BallsBowled.length === 0
                      ? "-"
                      : this.props.odiBowlerStats["0"].BallsBowled[0]
                          .player_stats_value
                    : console.log("not found")}
                </span>
                <span id="ball-stat-span-2">Balls bowled</span>
              </div>
              {/* <div className="bowling-stats-info-div">
                <span id="ball-stat-span-1">123.4</span>
                <span id="ball-stat-span-2">Economy rate</span>
              </div> */}
            </div>
            <div
              className="bowling-stats-info-parent-div"
              style={{ borderBottom: "none" }}
            >
              <div className="bowling-stats-bottom-info-div">
                <span id="ball-stat-span-1">
                  {this.props.odiBowlerStats["0"].TotalWIckets
                    ? this.props.odiBowlerStats["0"].TotalWIckets.length === 0
                      ? "-"
                      : this.props.odiBowlerStats["0"].TotalWIckets[0]
                          .player_stats_value
                    : console.log("not found")}
                </span>
                <span id="ball-stat-span-2">Wickets</span>
              </div>
              <div className="bowling-stats-bottom-info-div">
                <span id="ball-stat-span-1">
                  {this.props.odiBowlerStats["0"].EconomyRate
                    ? this.props.odiBowlerStats["0"].EconomyRate.length === 0
                      ? "-"
                      : this.props.odiBowlerStats["0"].EconomyRate[0]
                          .player_stats_value
                    : console.log("not found")}
                </span>
                <span id="ball-stat-span-2">Economy rate</span>
              </div>
              {/* <div className="bowling-stats-bottom-info-div">
                <span id="ball-stat-span-1">30.54</span>
                <span id="ball-stat-span-2">Strike Rate</span>
              </div> */}
              <div className="bowling-stats-bottom-info-div">
                <span id="ball-stat-span-1">
                  {this.props.odiBowlerStats["0"].FourWickets
                    ? this.props.odiBowlerStats["0"].FourWickets.length === 0
                      ? "-"
                      : this.props.odiBowlerStats["0"].FourWickets[0]
                          .player_stats_value
                    : console.log("not found")}
                </span>
                <span id="ball-stat-span-2">4 wickets</span>
              </div>
              <div className="bowling-stats-bottom-info-div">
                <span id="ball-stat-span-1">
                  {this.props.odiBowlerStats["0"].FiveWickets
                    ? this.props.odiBowlerStats["0"].FiveWickets.length === 0
                      ? "-"
                      : this.props.odiBowlerStats["0"].FiveWickets[0]
                          .player_stats_value
                    : console.log("not found")}
                </span>
                <span id="ball-stat-span-2">5 wickets</span>
              </div>
            </div>
          </div>
          <div className="bowling-T20-div">
            <h3 id="bowling-format-heading">T20</h3>
            <div className="bowling-stats-info-parent-div">
              <div className="bowling-stats-info-div">
                <span id="ball-stat-span-1">
                  {this.props.t20BowlerStats["0"].Matches
                    ? this.props.t20BowlerStats["0"].Matches.length === 0
                      ? "-"
                      : this.props.t20BowlerStats["0"].Matches[0]
                          .player_stats_value
                    : console.log("not found")}
                </span>
                <span id="ball-stat-span-2">Matches</span>
              </div>
              <div className="bowling-stats-info-div">
                <span id="ball-stat-span-1">
                  {this.props.t20BowlerStats["0"].Innings
                    ? this.props.t20BowlerStats["0"].Innings.length === 0
                      ? "-"
                      : this.props.t20BowlerStats["0"].Innings[0]
                          .player_stats_value
                    : console.log("not found")}
                </span>
                <span id="ball-stat-span-2">Innings</span>
              </div>
              <div className="bowling-stats-info-div">
                <span id="ball-stat-span-1">
                  {this.props.t20BowlerStats["0"].RunsConceded
                    ? this.props.t20BowlerStats["0"].RunsConceded.length === 0
                      ? "-"
                      : this.props.t20BowlerStats["0"].RunsConceded[0]
                          .player_stats_value
                    : console.log("not found")}
                </span>
                <span id="ball-stat-span-2">Runs Conceded</span>
              </div>
              <div className="bowling-stats-info-div">
                <span id="ball-stat-span-1">
                  {this.props.t20BowlerStats["0"].BallsBowled
                    ? this.props.t20BowlerStats["0"].BallsBowled.length === 0
                      ? "-"
                      : this.props.t20BowlerStats["0"].BallsBowled[0]
                          .player_stats_value
                    : console.log("not found")}
                </span>
                <span id="ball-stat-span-2">Balls bowled</span>
              </div>
            </div>
            <div
              className="bowling-stats-info-parent-div"
              style={{ borderBottom: "none" }}
            >
              <div className="bowling-stats-bottom-info-div">
                <span id="ball-stat-span-1">
                  {this.props.t20BowlerStats["0"].TotalWIckets
                    ? this.props.t20BowlerStats["0"].TotalWIckets.length === 0
                      ? "-"
                      : this.props.t20BowlerStats["0"].TotalWIckets[0]
                          .player_stats_value
                    : console.log("not found")}
                </span>
                <span id="ball-stat-span-2">Wickets</span>
              </div>

              <div className="bowling-stats-bottom-info-div">
                <span id="ball-stat-span-1">
                  {this.props.t20BowlerStats["0"].EconomyRate
                    ? this.props.t20BowlerStats["0"].EconomyRate.length === 0
                      ? "-"
                      : this.props.t20BowlerStats["0"].EconomyRate[0]
                          .player_stats_value
                    : console.log("not found")}
                </span>
                <span id="ball-stat-span-2">Economy rate</span>
              </div>
              <div className="bowling-stats-bottom-info-div">
                <span id="ball-stat-span-1">
                  {this.props.t20BowlerStats["0"].FourWickets
                    ? this.props.t20BowlerStats["0"].FourWickets.length === 0
                      ? "-"
                      : this.props.t20BowlerStats["0"].FourWickets[0]
                          .player_stats_value
                    : console.log("not found")}
                </span>
                <span id="ball-stat-span-2">4 wickets</span>
              </div>
              <div className="bowling-stats-bottom-info-div">
                <span id="ball-stat-span-1">
                  {this.props.t20BowlerStats["0"].FiveWickets
                    ? this.props.t20BowlerStats["0"].FiveWickets.length === 0
                      ? "-"
                      : this.props.t20BowlerStats["0"].FiveWickets[0]
                          .player_stats_value
                    : console.log("not found")}
                </span>
                <span id="ball-stat-span-2">5 wickets</span>
              </div>
            </div>
          </div>
          {/* <div className="bowling-Ipl-div">
            <h3 id="bowling-format-heading">IPL</h3>
            <div className="bowling-stats-info-parent-div">
              <div className="bowling-stats-info-div">
                <span id="ball-stat-span-1">500</span>
                <span id="ball-stat-span-2">Matches</span>
              </div>
              <div className="bowling-stats-info-div">
                <span id="ball-stat-span-1">623</span>
                <span id="ball-stat-span-2">Innings</span>
              </div>
              <div className="bowling-stats-info-div">
                <span id="ball-stat-span-1">21756</span>
                <span id="ball-stat-span-2">Runs Conceded</span>
              </div>
              <div className="bowling-stats-info-div">
                <span id="ball-stat-span-1">37568</span>
                <span id="ball-stat-span-2">Balls bowled</span>
              </div>
              <div className="bowling-stats-info-div">
                <span id="ball-stat-span-1">123.4</span>
                <span id="ball-stat-span-2">Economy rate</span>
              </div>
            </div>
            <div
              className="bowling-stats-info-parent-div"
              style={{ borderBottom: "none" }}
            >
              <div className="bowling-stats-bottom-info-div">
                <span id="ball-stat-span-1">375</span>
                <span id="ball-stat-span-2">Wickets</span>
              </div>
              <div className="bowling-stats-bottom-info-div">
                <span id="ball-stat-span-1">125</span>
                <span id="ball-stat-span-2">Average</span>
              </div>
              <div className="bowling-stats-bottom-info-div">
                <span id="ball-stat-span-1">30.54</span>
                <span id="ball-stat-span-2">Strike Rate</span>
              </div>
              <div className="bowling-stats-bottom-info-div">
                <span id="ball-stat-span-1">55</span>
                <span id="ball-stat-span-2">4 wickets</span>
              </div>
              <div className="bowling-stats-bottom-info-div">
                <span id="ball-stat-span-1">3</span>
                <span id="ball-stat-span-2">5 wickets</span>
              </div>
            </div>
          </div> */}
        </div>
      </div>
    );
  }
}

const mapStateToProps = state => ({
  odiBowlerStats: state.PlayerReducer.odiBowlerStats,
  t20BowlerStats: state.PlayerReducer.t20BowlerStats,
  testBowlerStats: state.PlayerReducer.testBowlerStats
});

export default connect(
  mapStateToProps,
  {
    getODIBowlerStats,
    getT20BowlerStats,
    getTestBowlerStats
    // getTestBatsmanStats,
    // getT20BatsmanStats,
    // getBatsmanStats
  }
)(PlayerBowlingStats);
