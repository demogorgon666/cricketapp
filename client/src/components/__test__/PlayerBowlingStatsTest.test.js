import React from "react";
import { shallow, mount } from "enzyme";
import { PlayerBowlingStats } from "../PlayerBowlingStats";

const getTestBowlerStats = jest.fn();
const getODIBowlerStats = jest.fn();
const getT20BowlerStats = jest.fn();
// const getTestBatsmanStats = jest.fn();
const testBowlerStats = [[]];
const t20BowlerStats = [[]];
const odiBowlerStats = [[]];

const wrapper = shallow(
  <PlayerBowlingStats
    getTestBowlerStats={getTestBowlerStats}
    getODIBowlerStats={getODIBowlerStats}
    getT20BowlerStats={getT20BowlerStats}
    // getTestBatsmanStats={getTestBatsmanStats}
    testBowlerStats={testBowlerStats}
    t20BowlerStats={t20BowlerStats}
    odiBowlerStats={odiBowlerStats}
  />
);

describe("Testing player Bowling stats ", () => {
  it("should render the componendt", () => {
    expect(wrapper).toMatchSnapshot();
  });
  it("should Have 4 headings Test,ODI,T20,IPL", () => {
    expect(
      wrapper
        .find("h3")
        .at(0)
        .text()
    ).toBe("Test");
    expect(
      wrapper
        .find("h3")
        .at(1)
        .text()
    ).toBe("ODI");
    expect(
      wrapper
        .find("h3")
        .at(2)
        .text()
    ).toBe("T20");
  });
});
