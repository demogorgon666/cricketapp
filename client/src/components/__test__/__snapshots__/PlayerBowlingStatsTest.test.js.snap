// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing player Bowling stats  should render the componendt 1`] = `
<div
  className="ball-main"
>
  <div
    className="bowling-main-div"
  >
    <div
      className="bowling-Test-div"
    >
      <h3
        id="bowling-format-heading"
      >
        Test
      </h3>
      <div
        className="bowling-stats-info-parent-div"
      >
        <div
          className="bowling-stats-info-div"
        >
          <span
            id="ball-stat-span-1"
          />
          <span
            id="ball-stat-span-2"
          >
            Matches
          </span>
        </div>
        <div
          className="bowling-stats-info-div"
        >
          <span
            id="ball-stat-span-1"
          />
          <span
            id="ball-stat-span-2"
          >
            Innings
          </span>
        </div>
        <div
          className="bowling-stats-info-div"
        >
          <span
            id="ball-stat-span-1"
          />
          <span
            id="ball-stat-span-2"
          >
            Runs Conceded
          </span>
        </div>
        <div
          className="bowling-stats-info-div"
        >
          <span
            id="ball-stat-span-1"
          />
          <span
            id="ball-stat-span-2"
          >
            Balls bowled
          </span>
        </div>
      </div>
      <div
        className="bowling-stats-info-parent-div"
        style={
          Object {
            "borderBottom": "none",
          }
        }
      >
        <div
          className="bowling-stats-bottom-info-div"
        >
          <span
            id="ball-stat-span-1"
          />
          <span
            id="ball-stat-span-2"
          >
            Wickets
          </span>
        </div>
        <div
          className="bowling-stats-bottom-info-div"
        >
          <span
            id="ball-stat-span-1"
          />
          <span
            id="ball-stat-span-2"
          >
            Economy rate
          </span>
        </div>
        <div
          className="bowling-stats-bottom-info-div"
        >
          <span
            id="ball-stat-span-1"
          />
          <span
            id="ball-stat-span-2"
          >
            4 wickets
          </span>
        </div>
        <div
          className="bowling-stats-bottom-info-div"
        >
          <span
            id="ball-stat-span-1"
          />
          <span
            id="ball-stat-span-2"
          >
            5 wickets
          </span>
        </div>
      </div>
    </div>
    <div
      className="bowling-Odi-div"
    >
      <h3
        id="bowling-format-heading"
      >
        ODI
      </h3>
      <div
        className="bowling-stats-info-parent-div"
      >
        <div
          className="bowling-stats-info-div"
        >
          <span
            id="ball-stat-span-1"
          >
             
          </span>
          <span
            id="ball-stat-span-2"
          >
            Matches
          </span>
        </div>
        <div
          className="bowling-stats-info-div"
        >
          <span
            id="ball-stat-span-1"
          />
          <span
            id="ball-stat-span-2"
          >
            Innings
          </span>
        </div>
        <div
          className="bowling-stats-info-div"
        >
          <span
            id="ball-stat-span-1"
          />
          <span
            id="ball-stat-span-2"
          >
            Runs Conceded
          </span>
        </div>
        <div
          className="bowling-stats-info-div"
        >
          <span
            id="ball-stat-span-1"
          />
          <span
            id="ball-stat-span-2"
          >
            Balls bowled
          </span>
        </div>
      </div>
      <div
        className="bowling-stats-info-parent-div"
        style={
          Object {
            "borderBottom": "none",
          }
        }
      >
        <div
          className="bowling-stats-bottom-info-div"
        >
          <span
            id="ball-stat-span-1"
          />
          <span
            id="ball-stat-span-2"
          >
            Wickets
          </span>
        </div>
        <div
          className="bowling-stats-bottom-info-div"
        >
          <span
            id="ball-stat-span-1"
          />
          <span
            id="ball-stat-span-2"
          >
            Economy rate
          </span>
        </div>
        <div
          className="bowling-stats-bottom-info-div"
        >
          <span
            id="ball-stat-span-1"
          />
          <span
            id="ball-stat-span-2"
          >
            4 wickets
          </span>
        </div>
        <div
          className="bowling-stats-bottom-info-div"
        >
          <span
            id="ball-stat-span-1"
          />
          <span
            id="ball-stat-span-2"
          >
            5 wickets
          </span>
        </div>
      </div>
    </div>
    <div
      className="bowling-T20-div"
    >
      <h3
        id="bowling-format-heading"
      >
        T20
      </h3>
      <div
        className="bowling-stats-info-parent-div"
      >
        <div
          className="bowling-stats-info-div"
        >
          <span
            id="ball-stat-span-1"
          />
          <span
            id="ball-stat-span-2"
          >
            Matches
          </span>
        </div>
        <div
          className="bowling-stats-info-div"
        >
          <span
            id="ball-stat-span-1"
          />
          <span
            id="ball-stat-span-2"
          >
            Innings
          </span>
        </div>
        <div
          className="bowling-stats-info-div"
        >
          <span
            id="ball-stat-span-1"
          />
          <span
            id="ball-stat-span-2"
          >
            Runs Conceded
          </span>
        </div>
        <div
          className="bowling-stats-info-div"
        >
          <span
            id="ball-stat-span-1"
          />
          <span
            id="ball-stat-span-2"
          >
            Balls bowled
          </span>
        </div>
      </div>
      <div
        className="bowling-stats-info-parent-div"
        style={
          Object {
            "borderBottom": "none",
          }
        }
      >
        <div
          className="bowling-stats-bottom-info-div"
        >
          <span
            id="ball-stat-span-1"
          />
          <span
            id="ball-stat-span-2"
          >
            Wickets
          </span>
        </div>
        <div
          className="bowling-stats-bottom-info-div"
        >
          <span
            id="ball-stat-span-1"
          />
          <span
            id="ball-stat-span-2"
          >
            Economy rate
          </span>
        </div>
        <div
          className="bowling-stats-bottom-info-div"
        >
          <span
            id="ball-stat-span-1"
          />
          <span
            id="ball-stat-span-2"
          >
            4 wickets
          </span>
        </div>
        <div
          className="bowling-stats-bottom-info-div"
        >
          <span
            id="ball-stat-span-1"
          />
          <span
            id="ball-stat-span-2"
          >
            5 wickets
          </span>
        </div>
      </div>
    </div>
  </div>
</div>
`;
